projects/ancestor/ancestor.py

```diff
@@ -3,35 +3,39 @@ def earliest_ancestor(ancestors, starting_node):
     paths = []
     visited = set()
 
+    # build our graph using children as keys
```

Comment:
Clever to use a set as Graph, But preferable using existing Graph class for this challenge